dftd3/dftd3.py

```diff
@@ -275,6 +275,38 @@ def d3(
     return attractive_r6_vdw + attractive_r8_vdw + repulsive_abc
 
 
+def D3_element_wise(elements, config, charges, *coordinates):
+    """Driver for the calculation of chosen derivatives to arbitrary order.
+
+    Parameters
+    ----------
+    elements : tuple
```

Comment:
Just a documentation update: I think it's best to pass a list of 2-tuples, rather than a tuple of tuples or an arbitrary length tuple.